en/docs/cli/add.md

```diff
@@ -48,7 +48,8 @@ You can also specify packages from different locations:
 3. `yarn add file:/path/to/local/tarball.tgz` installs a package from a gzipped
   tarball which could be used to share a package before publishing it.
 4. `yarn add <git remote url>` installs a package from a remote git repository.
-5. `yarn add https://my-project.org/package.tgz` installs a package from a
+5. `yarn add <git remote url>#<commit-ish>` installs a package from a remote git repository at specific git commit or git tag.
```

Comment:
We should just `commit/tag` instead of `commit-ish`

Could we just modify 4 to be `yarn add <git remote url>(#<commit/tag>)` or is that more confusing.